storage/oasis/nodeapi/file/node.go

```diff
@@ -0,0 +1,293 @@
+package file
+
+import (
+	"bytes"
+	"context"
+	"encoding/gob"
+	"encoding/json"
+	"fmt"
+
+	"github.com/akrylysov/pogreb"
+	beacon "github.com/oasisprotocol/oasis-core/go/beacon/api"
+	coreCommon "github.com/oasisprotocol/oasis-core/go/common"
+	consensus "github.com/oasisprotocol/oasis-core/go/consensus/api"
+	genesis "github.com/oasisprotocol/oasis-core/go/genesis/api"
+
+	"github.com/oasisprotocol/oasis-indexer/storage/oasis/nodeapi"
+	"github.com/oasisprotocol/oasis-indexer/storage/oasis/nodeapi/damask"
+)
+
+// FileConsensusApiLite provides access to the consensus API of an Oasis node.
+// Since FileConsensusApiLite is backed by a file containing the cached responses
+// to `ConsensusApiLite` calls, this data is inherently compatible with the
+// current indexer and can thus handle heights from both Cobalt/Damask.
+type FileConsensusApiLite struct {
+	db        pogreb.DB
+	damaskApi *damask.DamaskConsensusApiLite
```

Comment:
Prefer the `ConsensusApiLite` interface, not the specific `DamaskConsensusApiLite` impl. This way, we can back this cache with the upcoming "universal" ApiLite that works for all heights (by switching between Cobalt and Damask)

Reply:
Good catch, I'll update. Thanks!